### PR TITLE
Update swagger config to support getById, getAll, update, and delete for merchant redirect QRs

### PR DESCRIPTION
**Pull request by kaffikaffi**
This PR adds some new types, and configurations for endpoints regarding getById, getAll, update, and delete for merchant redirect QRs. 

This makes it possible to get all QRs from one saleunit, and the response will be like this:
```json
[
  {
    "id": "billboard_1",
    "url": "https://qr.vipps.no/generate/qr.png?..."
  },
  {
    "id": "billboard_2",
    "url": "https://qr.vipps.no/generate/qr.png?..."
  }
]
```
It also makes it possible to get by id, and update, but the response is the same as the post-endpoint.
Delete will return HTTP status code.

### CODE REVIEW

File: docs/swagger.yaml [hunk 1]
```diff
@@ -82,6 +82,96 @@ paths:
...
+      summary: Get all QRs
+      operationId: GetAllMerchantRedirectQrs
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MerchantRedirectQrResponseGetAll'
+        '400':
```
Comment (khenrichsen) [inline docs/swagger.yaml:95]: use the same schema for error as is used by the other endpoints

File: docs/swagger.yaml [hunk 2]
```diff
@@ -82,6 +82,96 @@ paths:
...
+        - $ref: '#/components/parameters/Ocp-Apim-Subscription-Key'
+    parameters:
+      - schema:
+          type: string
+        name: id
+        in: path
+        required: true
+        description: id for QR
+    get:
+      summary: Get all saleunit QRs
```
Comment (khenrichsen) [inline docs/swagger.yaml:156]: "Get all Merchant Redirect QRs"

File: docs/swagger.yaml [hunk 3]
```diff
@@ -82,6 +82,96 @@ paths:
...
+        '401':
+          description: Unauthorized
+      description: Get all QRs created for one saleunit
+      tags:
+        - Merchant redirect QR
+      parameters:
+        - $ref: '#/components/parameters/Authorization'
+        - $ref: '#/components/parameters/Ocp-Apim-Subscription-Key'
+  '/v1/merchantRedirect/{id}':
+    put:
```
Comment (khenrichsen) [inline docs/swagger.yaml:106]: Should we consider just having one get-endpoint, and accept Id or a list of id as a query parameter? Thats a pretty RESTful and normal way to handle get endpoints

File: docs/swagger.yaml [hunk 4]
```diff
@@ -82,6 +82,96 @@ paths:
...
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MerchantRedirectQrResponse'
+        '401':
+          description: Unauthorized
+        '404':
+          description: Not Found
+      description: Update redirectUrl
```
Comment (khenrichsen) [inline docs/swagger.yaml:120]: "Update the redirect url (target destination) of the QR"

**Review by khenrichsen — CHANGES_REQUESTED** [review]

**Review by Sigvedash — APPROVED** [review]

File: docs/swagger.yaml (hunk 2 above)
Comment (khenrichsen) [reply docs/swagger.yaml]: This wasnt added on all endpoints @kaffikaffi 
image
its missing on delete and Get endpoints

**Pull request merged**